docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -68,12 +70,34 @@ kubectl edit -n capi-system deployment.apps/capi-controller-manager
       --feature-gates=MachinePool=true,ClusterResourceSet=true
 ```
 
-Similarly, to **validate** if a particular feature is enabled, see cluster-api-provider deployment arguments by:
+Similarly, to **validate** if a particular feature is enabled, see the arguments by issuing:
 
 ```bash
 kubectl describe -n capi-system deployment.apps/capi-controller-manager
 ```
 
+Following controller manager deployments have to be edited in order to enable/disable their respective experimental features:
+
+* [MachinePools](./machine-pools.md):
+  * [CAPI](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#capi).
+  * [CABPK](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#cabpk).
+  * [CAPD](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Providers#capd). Other [Infrastructure Providers](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Providers#infrastructure-provider)
+    might also require this. Please consult the docs of the concrete [Infrastructure Provider](https://cluster-api.sigs.k8s.io/reference/providers#infrastructure)
+    regarding this.
+* [ClusterResourceSet](./cluster-resource-set.md):
+  * [CAPI](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#capi).
+* [ClusterClass](./cluster-class/index.md):
+  * [CAPI](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#capi).
+  * [KCP](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#kcp).
+* [Ignition Bootstrap configuration](./ignition.md):
+  * [CABPK](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#cabpk).
+  * [KCP](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#kcp).
+* [Runtime SDK](runtime-sdk/index.md):
+  * [CAPI](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#capi).
+  * [CAPD](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Providers#capd). Other [Infrastructure Providers](https://cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Providers#infrastructure-provider)
```

Comment:
This should be added to ClusterClass instead

Reply:
Oh, good catch. Wrong copy paste :).